src/data/javascript-quiz.ts

```diff
@@ -260,6 +260,20 @@ const javascriptQuiz = [
       "Assigning 0 to the length property of the array will clear the contents of the array.",
     Link: "https://www.freecodecamp.org/news/how-javascript-implements-oop/"
   },
+  {
+    Question:
+      "In Javascript class-based OOP, are objects and classes the same constructs?",
```

Comment:
the word JavaScript needs to be spelled correctly here 

```suggestion
      "In JavaScript class-based OOP, are objects and classes the same constructs?",
```